web-registry/src/lib/ecocredit/ecocredit.config.ts

```diff
@@ -0,0 +1,10 @@
+export const v1AlphaBatchDenomMapping: Record<string, string> = {
```

Comment:
could you add a comment explaining why we need this hardcoded mapping?

Reply:
Will do.

Reply:
Done.